.travis.yml

```diff
@@ -32,20 +32,20 @@ matrix:
       script:
         - ./functional.sh verbose
     - stage: build
-      name: "Unit Tests: Python 2.7 with Attrs 18 and 19"
```

Comment:
I realized there was a bug here, the comment said we were testing with attrs 19 but we weren't. I've fixed it and included 2 more versions of attrs.